deploy/org.mavlink.qgroundcontrol.metainfo.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.mavlink.qgroundcontrol</id>
+  
+  <name>QGroundControl</name>
+  <summary>UAS ground control station</summary>
+  
+  <metadata_license>MIT</metadata_license>
```

Comment:
Maybe this needs discussion still. Within the generator website, I couldn't choose either GPL and/or Apache. :confused:

Reply:
Found a solution to declare GPL3 and Apache2.